docs/UserGuide.md

```diff
@@ -34,7 +34,7 @@ PayMeLah is a **desktop app for managing the debts your friends owe you, optimiz
 
    * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+6. Refer to the [Features](#features) below for details of each command.
 
```

Comment:
Should revert the numbering back to 1s, as per the 2103 style guide for GFMD.

Reply:
Changed to generic numbering as requested!